docs/api.md

```diff
@@ -0,0 +1,90 @@
+# Bus Time Backend API
```

Comment:
Not sure I fully understand. Do you suggest renaming `api.md`, `heroku-running.md`, `local-running.md` to `API.md`, `Heroku Running.md`, `Local Running.md` respectively?

If so, I would prefer to stick with lowercase hyphen-delimited naming like all other files are named. Easier to type in terminal and it looks more familiar as a part of a URL.